src/scripts/main.js

```diff
@@ -2,7 +2,26 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+for (const person of people) {
+  const tableRow = document.createElement('tr');
 
-// write your code here
+  const fullName = document.createElement('td');
+  const sex = document.createElement('td');
+  const born = document.createElement('td');
+  const died = document.createElement('td');
+  const age = document.createElement('td');
+  const century = document.createElement('td');
+
+  fullName.innerHTML = person.name;
+  sex.innerText = person.sex;
+  born.innerText = person.born;
+  died.innerHTML = person.died;
+  age.innerText = person.died - person.born;
+  century.innerText = Math.ceil(person.died / 100);
+
+  tableRow.append(fullName, sex, born, died, age, century);
```

Comment:
or you can use insertAdjacentHTML and add all this rows as HTML
like this ` <td>${person.name}</td>`